build/gn_run_malioc.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I'm a bit uncertain about the need for this wrapper script since the script that it wraps is already a wrapper around the underlying malioc binary. Is there some way to merge this functionality into the existing wrapper script in the engine repo?

Reply:
Oh shoot, I should have read malioc.gni more closely. I mistakenly thought it was running malioc directly.